### PR TITLE
Makes logging consistent in all modules in `webotsrobocup` role

### PR DESCRIPTION
**Pull request by KipHamiltons**

### CODE REVIEW

**Review by NDxiura — COMMENTED** [review]
Only one of the three QuinticWalk.yaml files have log_level: WARN applied. (Only this one is used in the current code)

File: module/support/logging/DataLogging/src/DataLogging.cpp [hunk 1]
```diff
@@ -112,7 +112,9 @@ namespace module::support::logging {
 
         on<Configuration, Trigger<CommandLineArguments>, Sync<DataLog>>("DataLogging.yaml")
             .then([this](const Configuration& cfg, const CommandLineArguments& argv) {
-                {
+                {  // TODO(KipHamiltons): Clarify if this scope even does anything...
+                    log_level = cfg["log_level"].as<NUClear::LogLevel>();
+
```
Comment (tayloryoung6396) [inline module/support/logging/DataLogging/src/DataLogging.cpp:117]: Make an issue for this TODO

**Review by tayloryoung6396 — CHANGES_REQUESTED** [review]

File: module/support/logging/DataLogging/src/DataLogging.cpp (hunk 1 above)
Comment (KipHamiltons) [reply module/support/logging/DataLogging/src/DataLogging.cpp:117]: Done

**Comment by KipHamiltons** [comment]
Tested and working as of right now

**Review by tayloryoung6396 — DISMISSED** [review]

**Comment by KipHamiltons** [comment]
Let's get this merged

**Review by JosephusPaye — APPROVED** [review]
LGTM

**Pull request merged**